fdbserver/ptxn/TLogPeekCursor.actor.h

```diff
@@ -513,9 +513,10 @@ class BroadcastedStorageTeamPeekCursor_Ordered : public BroadcastedStorageTeamPe
 	virtual void nextImpl() override;
 
 public:
-	BroadcastedStorageTeamPeekCursor_Ordered()
+	BroadcastedStorageTeamPeekCursor_Ordered(const Version& version_ = 0)
```

Comment:
why do we want to have a default here, as the base class already have a default?

Reply:
Because the default value cannot be forwarded to the base class automatically. It has to be explicitly set and forwarded.